lib/get-dialect-ids.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { test, expect, describe } from "vitest";
+import { getDialectIds, loadDialect } from "./keywords.js";
```

Comment:
In tests I try to import only from public APIs. This tests that the function was exported properly.

```suggestion
import { getDialectIds, loadDialect } from "./experimental.js";
```